data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,288 @@ mission "Arachi Cartoonist 2"
+			`	It looks like Ulmodi finished fourth; he's up next to be interviewed. The event seems to be over, and the crowd disperses under the supervision of Heliarch agents. Instead of heading back to the train, they're moving to a different section of the oddly-shaped stadium.`
+			`	You go with the crowd and are shown to a room with many elevators. Eventually, everyone descends to the valley below and follows a path to an actual, full-size stadium, and a massive one at that.`
+			`	As you sit among the alien beings feeling a little out of place, you realize you're probably the first human ever to witness these games. You look over the stadium grounds. It's rectangular, and small mounds dot the field. Six large holes are arranged evenly on the sides of the field.`
+			`	Half an hour passes, and a Kimek sitting next to you with a translation device explains that the contestants are resting. A couple more minutes pass before the thirty Saryds, holding what looks like fancy polo mallets, enter the grounds and are divided into six teams of different colors. You see that Ulmodi is on the blue team. Several large spheres are placed at the center of the stadium in a circle, and one member from each team stands around them.`
+			`	The game starts. It's some sort of mix between polo, billiards, and golf. The teams trot around for a while, and rapidly rush to the spheres once they see an opening in the others' defenses. As they hit the spheres with their mallets, they roll around the mounds, curving all over and sometimes even accidentally being flung into the audience; fortunately, nets are in place to keep the spheres within the playing area.`
+			`	Eventually, a sphere finds its way into a hole, which lights up with the color of the team who scored: red. Much of the crowd jumps up and screams. You're not sure if everyone is a fan of the red team or if they're cheering simply because a point was scored. The sphere is brought back up automatically via a tube close to the hole and it's back in the game.`
+			`	The game continues, and each team accumulates a few points. The blue team, leading, scores a fifth point, and the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The Kimek with the translator explains what is happening.`
+			`	"The winning score, they have reached! The winners, the blue team are!" They say, still in a cheering mood. After everyone settles down, the members of the teams are shown on the replay screen, going from the team that scored the least all the way to the blue team at the end. Ulmodi is placed third, which must mean a lot considering him and the other two above him celebrate by running laps around the inside of the stadium.`
```

Comment:
Simple "the" to "a," as "the" implies to me that the screen was previously introduced. I'd also just add a small snippet about how Ulmodi is placed in third because of his performance in both events.
```suggestion
			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, going from the team that scored the least all the way to the blue team at the end. Between the two events, Ulmodi is placed third, which must mean a lot considering him and the other two above him celebrate by running laps around the inside of the stadium.`
```